modules/statistics/templates/form_stats_behavioural.tpl

```diff
@@ -48,7 +48,7 @@
             <td class="   pis">Per Instrument Stats</td>
             {foreach from=$Centers item=center key=centername}
                 <td id='{$center.ID}PIS' class="pis" colspan="2">
-                    <a href='{$baseurl}/statistics/statistics_site/?CenterID={$center.NumericID}{if $useProjects == "true"}&ProjectID={$CurrentProject.ID}{/if}' target="_blank">Please Click Here</a>
+                    <a href='{$baseurl}/statistics/statistics_site/?CenterID={$center.NumericID}{if $useProjects == "true"}&ProjectID={$CurrentProject.ID}{/if}' target="_blank">View Details</a>
```

Comment:
@ZainVirani What insipired this change ?

Reply:
`Please Click Here` doesn't really mean anything. It doesn't specify what clicking `here` actually does. Redirect? Refresh? Show more data?
`View Details` is way more clear

Reply:
makes sense, just wondering if it was a request from someone in particular